testsuite/features/secondary/min_change_software_channel.feature

```diff
@@ -1,10 +1,10 @@
 # Copyright (c) 2020 SUSE LLC
 # Licensed under the terms of the MIT license.
 
-Feature: Assign child channel to system
+Feature: Assign child channel to a system
```

Comment:
What do you think about the filename? Should the filename reflect the VM we are using to run that test? Or should the filename reflect what the feature title is about? :thinking:

Reply:
The VM, definitely. Using "a system" is just a trick to avoid duplicate scenario names.